packages/electrode-archetype-react-app/config/webpack/webpack.config.hot.js

```diff
@@ -18,6 +20,9 @@ var config = module.exports = _.flow(
   hotConfig()
 )();
 
+
+config.devServer = {}; // use webpack default verbosity
```

Comment:
This actually causes hot mode to not work. The internal and external archetypes are a bit different in how they run their dev servers. It looks like removing this line fixes the issue. We might need to dive into the exact params to get "default verbosity" if it is important.